### PR TITLE
* Fix typos

### PR DESCRIPTION
**Pull request by Wagnerp**
* Fix typos

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Utilities/IconExtractor.cs [hunk 1]
```diff
@@ -1,8 +1,14 @@
-﻿// *****************************************************************************
-// BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit)
-// by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2022. All rights reserved. 
-//  Version 6.0.0  
-// *****************************************************************************
+﻿#region BSD License
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Utilities/IconExtractor.cs:1]: These are new files and have nothing to do with the original, hence not included

File: Source/Krypton Components/Krypton.Toolkit/Utilities/IconUtilities.cs [hunk 2]
```diff
@@ -1,13 +1,20 @@
-﻿// *****************************************************************************
-// BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit)
-// by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2022. All rights reserved. 
-//  Version 6.0.0  
-// *****************************************************************************
+﻿#region BSD License
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Utilities/IconUtilities.cs:1]: These are new files and have nothing to do with the original, hence not included

File: Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs [hunk 3]
```diff
@@ -1,8 +1,14 @@
-﻿// *****************************************************************************
-// BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit)
-// by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2022. All rights reserved. 
-//  Version 6.0.0  
-// *****************************************************************************
+﻿#region BSD License
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs:1]: These are new files and have nothing to do with the original, hence not included

File: Source/Krypton Components/Krypton.Toolkit/Utilities/PaletteImageScaler.cs [hunk 4]
```diff
@@ -38,14 +38,12 @@ public static class PaletteImageScaler
         /// <param name="pal">KryptonPalette</param>
         public static void ScalePalette(float factorDpiX, float factorDpiY, KryptonPalette pal)
         {
-            if (pal == null
-//            || pal.HasAlreadyBeenScaled
-                )
+            if (pal == null /*|| pal.HasAlreadyBeenScaled*/)
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Utilities/PaletteImageScaler.cs:41]: keep conditions on separate lines, it is then easier to comment out, put back in to test

File: Source/Krypton Components/Krypton.Toolkit/Utilities/RoundedRectangleF.cs [hunk 5]
```diff
@@ -18,21 +18,28 @@ namespace Krypton.Toolkit
     public class RoundedRectangleF
     {
         #region Variables
-        private readonly float _x;
-        private readonly float _y;
-        private readonly float _width;
-        private readonly float _height;
+        private readonly float _x, _y, _width, _height;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Utilities/RoundedRectangleF.cs:21]: This is a bad practice patter.. Please undo.

File: Source/Krypton Components/Krypton.Toolkit/Utilities/IconExtractor.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/IconExtractor.cs]: Done

File: Source/Krypton Components/Krypton.Toolkit/Utilities/IconUtilities.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/IconUtilities.cs]: Done

File: Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs]: Done

File: Source/Krypton Components/Krypton.Toolkit/Utilities/PaletteImageScaler.cs (hunk 4 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/PaletteImageScaler.cs]: Done

File: Source/Krypton Components/Krypton.Toolkit/Utilities/RoundedRectangleF.cs (hunk 5 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/RoundedRectangleF.cs]: Done

File: Source/Krypton Components/Krypton.Toolkit/Utilities/IconExtractor.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/IconExtractor.cs]: Reverted

File: Source/Krypton Components/Krypton.Toolkit/Utilities/IconUtilities.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/IconUtilities.cs]: Reverted

File: Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs]: Reverted

**Pull request merged**